src/ufo-general-backproject-task.c

```diff
@@ -284,6 +284,7 @@ struct _UfoGeneralBackprojectTaskPrivate {
     gdouble gray_map_min, gray_map_max;
     /* Private */
     guint count, generated;
+    UfoResources *resources;
```

Comment:
@matze here we go. In this an the last commit I created device-specific build options. Is it OK with the structure and the hash table or do you have a better idea? Thanks.

Reply:
Why are you making the table static and not part of the private structure? This looks lika a classic race condition to me with multiple threads accessing that table.

Reply:
First it wasn't, every node created it's own, but then I thought we don't need to recreate that all the time (not that it would take too much time for that tiny structure, but still). If it's in the private structure and initialized in setup then it will be also initialized as many times as the node is copied, or not?

Reply:
>  If it's in the private structure and initialized in setup then it will be also initialized as many times as the node is copied, or not?

Correct. But as you said yourself, it's not much data anyway and I'd rather like to see it working than crashing ;-)

Reply:
Done in 643da0019583319d8cbcab5633f0fe9fb8abfaee